onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -1412,6 +1414,15 @@ def aten_cartesian_prod(tensors: Sequence[TensorType]) -> TensorType:
     raise NotImplementedError()
 
 
+@torch_op("aten::cat", trace_only=True, complex=True)
+def aten_cat_complex(tensors: Sequence[TTensor], dim: int = 0) -> TTensor:
```

Comment:
We can test this I think?

Reply:
Not very meaningful. If we want to test this, the input generated in `extra_opinfo` will be real representation of the s. The crucial part is actually during the conversion.

Reply:
Simply adding an entry with complex=True should do it:

https://github.com/microsoft/onnxscript/blob/635fb437f8270769b1d19b3e286384d7f80a2741/onnxscript/tests/function_libs/torch_lib/ops_test_data.py#L534-L535

Reply:
It doesn't work, because the tests are like `input=(-5), kwargs={'dtype':torch.complex64}`, but `aten_scalar_tensor_complex` only supports `input=(-5j), kwargs={'dtype':torch.complex64}`. When it comes to scalar real input, we need the original one. It's special case.

Reply:
Can we filter those cases out?

Reply:
I can do that, but that's all of them.

Reply:
Add it for remind that is a special case.